guardian/commercial-templates
# remove ad label from ad templates

## What does this change?

This PR removes the insertion of ad Labels being rendered as part of the Native ad. 
We are removing it from source as we will now be rendering the ad label via the ad slots in commercial and DCR.

see related PR's : [commercial](https://github.com/guardian/commercial/pull/1484) and [DCR](https://github.com/guardian/dotcom-rendering/pull/12034)



## How to test
Checking out this branch and pnpm run dev and check: http://localhost:7777/csr/fabric

This was also tested with GAM line item:
https://admanager.google.com/59666047#delivery/line_item/detail/line_item_id=6756767022&li_tab=settings&sort_by=Status&sort_asc=false
You can add `?adtest=fluid-adlabel-hide` at the end of the URL

Creating a Native using the latest HTML and CSS resulting from local development and adding it to styles GAM Native Style [here](https://admanager.google.com/59666047#creatives/native/native_style/detail/style_id=869240&tab=ad_settings)

### **Important note**: Visual regression test failing:
`Expected an image 360px by 275px, received 360px by 251px. 14785 pixels (ratio 0.15 of all image pixels) are different.`
- This failure indicates the successful removal of the ad label.
- The "expected" image is dynamically generated by the Playwright test.
- This generated image is stored in the playwright/reference-images directory and reflects the current state of the source code. Result [here](https://guardian.github.io/commercial-templates/) 
- To update the reference image and align it with the recent changes, we need to merge the current branch into the main branch. This action will ensure the "expected" image is updated and future tests will be validated against the latest changes.




## How can we measure success?
We should hopefully not see any ad labels at the top of the add.


## Images

| Before. | After |
|-------| -------| 
|<img width="1205" alt="Screenshot 2024-07-31 at 15 02 10" src="https://github.com/user-attachments/assets/c121c2eb-6fd4-406c-bf06-cc0ade2567f8">|<img width="1099" alt="Screenshot 2024-07-31 at 15 00 22" src="https://github.com/user-attachments/assets/14f8771f-990f-44b3-839b-b3595d079a74">|




## Accessibility
